anchore_engine/subsys/events/factories.py

```diff
@@ -0,0 +1,57 @@
+from anchore_engine.subsys.events import UserAnalyzeImageCompleted
```

Comment:
@nightfurys I pulled these out so they'd 1) be in a common location between the services and 2) be more testable. I am not at all married to the module name if you have a better suggestion.

Reply:
Refactoring this for testability is fine by me, can we rename the file to utils?